.github/CONTRIBUTING.md

```diff
@@ -1,22 +1,27 @@
-## Contributing to GEF ##
+## Contributing to GEF
 
-### License ###
+## License
 
-`gef` is placed under [MIT license](https://github.com/hugsy/gef/blob/main/LICENSE) which provides Open-Source access to the code and its use.
+`gef` is placed under [MIT license](https://github.com/hugsy/gef/blob/main/LICENSE) which provides
+Open-Source access to the code and its use.
 
-By contributing to `gef` code through the _Pull Requests_ mechanism, you accept to release the code written by you under the said license.
+By contributing to `gef` code through the _Pull Requests_ mechanism, you accept to release the code
+written by you under the said license.
 
+## Submitting a Patch
 
-### Submitting a Patch ###
-
-1. Fork `gef` repository (requires GitHub account). Sending a patch from the
+1.  Fork `gef` repository (requires GitHub account). Sending a patch from the
    `patch` or `git diff --patch` commands is not accepted.
-1. Write the changes in your local repo.
-    * Respect the coding style (same indentation format, explicit names as possible)
-    * Comment your code
-    * If you add a new feature/GDB command, also write the adequate
-      documentation (in [`docs/`](docs/))
-1. Submit a pull request, **make sure it is made against the `dev` branch (not `main`, which only contains tested and stable features)**
-1. The contributors will review your patch. If it is approved, the change will
+1.  All the packages required for testing and documenting are listed in `tests/requirements.txt`
+1.  Adjust your development environment to GEF's: this is achieved using
+   [`pre-commit`](https://re-commit.com/), and getting setup is simply done by
```

Comment:
a little mistake:

https://pre-commit.com/

Reply:
Good catch
A PR is coming to check links in markdown so this won't go unnoticed in the future 😀

Reply:
Fixed by e2af9d2